setuptools/__init__.pyi

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Mapping, Optional, Sequence, Union
+
+import setuptools
+
+
+def setup(
+    name: str,
```

Comment:
Name is also optional (to supply). Consider [this usage](https://github.com/jaraco/skeleton/blob/main/setup.py), which I recommend. All parameters are specified in setup.cfg.

Reply:
That is an excellent point, I'll change this in a moment.